### PR TITLE
Add pharaoh's sceptres, tome of fire, and toxic staff of the dead item mappings

### PR DESCRIPTION
**Pull request by MagicfTail**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java [hunk 1]
```diff
@@ -190,7 +192,16 @@
 		BLACK_MASK, BLACK_MASK_I, BLACK_MASK_1, BLACK_MASK_1_I, BLACK_MASK_2, BLACK_MASK_2_I, BLACK_MASK_3, BLACK_MASK_3_I, BLACK_MASK_4, BLACK_MASK_4_I, BLACK_MASK_5,
 		BLACK_MASK_5_I, BLACK_MASK_6, BLACK_MASK_6_I, BLACK_MASK_7, BLACK_MASK_7_I, BLACK_MASK_8, BLACK_MASK_8_I, BLACK_MASK_9, BLACK_MASK_9_I, BLACK_MASK_10_I,
 		SLAYER_HELMET, SLAYER_HELMET_I, BLACK_SLAYER_HELMET, BLACK_SLAYER_HELMET_I, PURPLE_SLAYER_HELMET, PURPLE_SLAYER_HELMET_I, RED_SLAYER_HELMET, RED_SLAYER_HELMET_I,
-		GREEN_SLAYER_HELMET, GREEN_SLAYER_HELMET_I, TURQUOISE_SLAYER_HELMET, TURQUOISE_SLAYER_HELMET_I);
+		GREEN_SLAYER_HELMET, GREEN_SLAYER_HELMET_I, TURQUOISE_SLAYER_HELMET, TURQUOISE_SLAYER_HELMET_I),
+
+	//Pharaoh's Sceptres
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: space before //

Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: I have been wrong before, but it looks to me like none of the other comments in this file has a space before them

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: Oh, I meant space after // and before comment :d

Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: Oh, 🤦‍♂️

**Comment by deathbeam** [comment]
```diff
- //Pharaoh's Sceptres
+// Pharaoh's Sceptres
```

**Pull request merged**